docs/workflow.puml

```diff
@@ -1,26 +1,26 @@
 @startditaa
                          +----------------+
-                         |ilab init       |-=-taxonomy repo cloned           legend
+                         |ilab config init       |-=-taxonomy repo cloned           legend
```

Comment:
It looks like the changes throughout this file broke the spacing layout of these boxes.  It looks like a bunch of spaces need to be removed in the lines where you added words.

Reply:
Oh I see that now! Lemme fix that!

Reply:
Just updated!